app/server/tests/functions/postFunctions.js

```diff
@@ -0,0 +1,38 @@
+require('dotenv').config();
+const supertest = require('supertest');
+const endpoint = process.env.API_REF;
+const request = supertest(endpoint);
+
+
+function createPost(token){
+    return request
+        .post('/post')
+        .set({authorization: `Bearer ${token}`});
+} 
+
+function getAllPosts(token){
+    return request
+        .get(`/post`)
+        .set({authorization: `Bearer ${token}`});
+} 
+
+function getPost(id,token){
```

Comment:
```suggestion
function getPost(id, token){
```

Reply:
sorry I'm nitpicking here. is this acceptable?

Reply:
ya for sure, will try to keep on top of my spaces 👍